docs/templates/process-logging.md

```diff
@@ -0,0 +1,292 @@
+# Workspace Process Logging
+
+The workspace process logging feature allows you to log all system-level
+processes executing in the workspace.
+
+> **Note:** This feature is only available on Linux in Kubernetes. There are
+> additional requirements outlined further in this document.
+>
+> This is an Enterprise feature. To learn more about Coder Enterprise, please
+> contact sales.
+
+Workspace process logging adds a sidecar container to workspace pods that will
+log all processes started in the workspace container (e.g., commands executed in
+the terminal or processes created in the background by other processes).  You
```

Comment:
Should we also mention that commands executed within a container are captured For example someone could type `docker run -it /bin/sh` and then any commands inside that `sh` are also captured? Or do you think that will be obvious to most users?

Reply:
I added a sentence explaining that container and nested container processes are also logged.